src/modules/tasks.js

```diff
@@ -14,34 +35,54 @@ function addTaskToDOM(task) {
   }
   taskElement.dataset.id = task.id;
 
-  const titleElement = document.createElement('span');
-  titleElement.className = 'title';
-  titleElement.contentEditable = true;
-  titleElement.textContent = task.title;
-  titleElement.addEventListener('input', () => {
-    task.title = titleElement.textContent;
-    saveTasksToLocalStorage();
-  });
+  const titleElement = createTitleElement(task.title);
+  titleElement.addEventListener('input', handleTitleInput);
 
-  const deleteButton = document.createElement('span');
-  deleteButton.className = 'del';
-  deleteButton.appendChild(document.createTextNode('Delete'));
+  const deleteButton = createDeleteButton();
   taskElement.appendChild(titleElement);
   taskElement.appendChild(deleteButton);
   taskList.appendChild(taskElement);
+  // update the index prop of each remaining task
+  task.index = tasks.length;
```

Comment:
| After adding some todos | After deleting one todo |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/93039359/236634664-22e2d9fb-0e23-41c6-b5dc-483d1173ed48.png) | ![image](https://user-images.githubusercontent.com/93039359/236634693-28612393-546e-4c92-8d1e-ee2f92786c5c.png) |

- [ ] As you can see in the image above that when i created some items then the index started from 1 :+1: but when i deleted one todo then the index started from 0 :fearful: Kindly start it from 1 as well to make your project follow the guidelines as suggested by the previous reviewer :blush: